web/modules/custom/hoeringsportal_public_meeting/assets/js/single-date.js

```diff
@@ -0,0 +1,10 @@
+require('../css/single-date.css')
+
+document.querySelectorAll('.date-to-element input.form-date')[0].disabled = true
```

Comment:
Use [`document.querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) when you only want the first element.